homework_1/task1/fizzBuzz/fizzBuzz.go

```diff
@@ -0,0 +1,38 @@
+package fizzBuzz
```

Comment:
> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. They are often simple nouns

~ https://blog.golang.org/package-names

According to that convention `fizzbuzz` would be better than `fizzBuzz` here.